sdk/tables/Azure.Data.Tables/tests/samples/Sample1_CreateDeleteTable.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using Azure.Data.Tables.Tests;
+using Azure.Data.Tables.Models;
+
+namespace Azure.Data.Tables.Samples
+{
+    [LiveOnly]
+    public partial class TablesSamples : TablesTestEnvironment
+    {
+        [Test]
+        public void CreateTable()
+        {
+            string storageUri = StorageUri;
+            string accountName = AccountName;
+            string storageAccountKey = PrimaryStorageAccountKey;
+            string tableName = "OfficeSupplies";
+
+            #region Snippet:TablesSample1CreateClient
+            // Construct a new <see cref="TableServiceClient" /> using a <see cref="TableSharedKeyCredential" />.
+            var serviceClient = new TableServiceClient(
+                new Uri(storageUri),
+                new TableSharedKeyCredential(accountName, storageAccountKey));
+            #endregion
+
+            try
+            {
+                #region Snippet:TablesSample1CreateTable
+                // Create a new table. Getting response is not necessary to create; just gives more info.
+                TableItem response = serviceClient.CreateTable(tableName);
```

Comment:
nit: I think a more descriptive name would be nice for the `TableItem` here. ex `createdTable` or `table`.

Reply:
what do u mean with 
> Getting response is not necessary to create; just gives more info.

Reply:
> nit: I think a more descriptive name would be nice for the `TableItem` here. ex `createdTable` or `table`.

I think I wasn't sure what to call it because I didn't want the user to get confused by it being called `table`  to avoid it be confused as the table client.



> what do u mean with
> 
> > Getting response is not necessary to create; just gives more info.

This is where I was trying to explain that the user doesn't need to store the return value for any functional purpose.

Reply:
> what do u mean with
> 
> > Getting response is not necessary to create; just gives more info.

I ended up changing that comment to describe what TableItem is to distinguish it from TableClient!